docs/components/concepts/expressions.md

```diff
@@ -23,6 +23,8 @@ Additionally, the following attributes of BPMN elements can define an expression
 - User task: [assignee](/components/modeler/bpmn/user-tasks/user-tasks.md#assignments), [candidateGroups](/components/modeler/bpmn/user-tasks/user-tasks.md#assignments)
 - Call activity: [process id](/components/modeler/bpmn/call-activities/call-activities.md#defining-the-called-process)
 
+And many more.
```

Comment:
This is a bit vague. Can we link to these, or be more specific?

Reply:
True. :sweat_smile: 

Currently, the text mentions a few attributes but not all of them. I don't see value in having a complete list of attributes here. And, it is hard to maintain when adding new features. 

What do you about removing the whole list? 

Instead, we could mention one attribute as an example.

Reply:
I agree with this, can always link to additional documentation as well.